api/routes/employees.js

```diff
@@ -0,0 +1,14 @@
+const router = require('express').Router()
+const Employee = require('../models/employees')
```

Comment:
You've set Employee to just be a schema. Which, is fine! But it means you'll need to go through the model of `Unit` to get employees.